app.js

```diff
@@ -1499,6 +1499,162 @@ app.delete("/room/:roomId", function (req, res) {
     });
 });
 
+
+//suggest meeting times given list of participants
+app.post("/api/meetingSuggestion", function (req, res) {
+
+    //currently only look 3 days ahead, and assume 7AM - 5PM workday. only search on weekdays
```

Comment:
Let's encode these values as defaults, so they could theoretically be changed.

Reply:
Will do.